utils/create_aws_region_data.py

```diff
@@ -0,0 +1,60 @@
+from textwrap import dedent
+
+import boto3
+
+region_names = {
+    'us-east-1': 'US East (N. Virginia)',
+    'us-east-2': 'US East (Ohio)',
+    'us-west-1': 'US West (N. California)',
+    'us-west-2': 'US West (Oregon)',
+    'af-south-1': 'Africa (Cape Town)',
+    'ap-east-1': 'Asia Pacific (Hong Kong)',
+    'ap-south-1': 'Asia Pacific (Mumbai)',
+    'ap-northeast-3': 'Asia Pacific (Osaka)',
+    'ap-northeast-2': 'Asia Pacific (Seoul)',
+    'ap-southeast-1': 'Asia Pacific (Singapore)',
+    'ap-southeast-2': 'Asia Pacific (Sydney)',
+    'ap-northeast-1': 'Asia Pacific (Tokyo)',
+    'ca-central-1': 'Canada (Central)',
+    'eu-central-1': 'EU (Frankfurt)',
+    'eu-west-1': 'EU (Ireland)',
+    'eu-west-2': 'Europe (London)',
+    'eu-south-1': 'Europe (Milan)',
+    'eu-west-3': 'Europe (Paris)',
+    'eu-north-1': 'Europe (Stockholm)',
+    'me-south-1': 'Middle East (Bahrain)',
+    'sa-east-1': 'South America (São Paulo)',
+}
+
+
+def main():
+
+    for region in ['us-east-1', 'us-west-2', 'eu-west-1', 'eu-west-1', 'eu-west-2', 'eu-central-1']:
```

Comment:
Same here. Duplicated `eu-west-1` and missing `eu-north-1`.

Reply:
fixed